agent/consul/state/config_entry_exported_services_ce_test.go

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+//go:build !consulent
```

Comment:
Haven't seen the enterprise version of this PR? Is it available? 😅

Reply:
I have raised the enterprise PR as well. Will share it with you